packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesAppLoader.m

```diff
@@ -175,21 +175,26 @@ - (void)startLoadingFromManifest:(EXUpdatesUpdate *)updateManifest
         if (matchingAssetError || !matchingDbEntry || !matchingDbEntry.filename) {
           [self downloadAsset:asset];
         } else {
+          EXUpdatesAsset *assetToLoad = asset;
```

Comment:
```suggestion
          EXUpdatesAsset *assetToLoad;
```
1. Would be a bit easier to read if assetToLoad was assigned only in the if/else statement.
2. Also maybe put this declaration right above the if/else on line 181.

Reply:
(if there are nullability complaints then no worries about the (1).